content/using/id/overview.md

```diff
@@ -7,65 +7,189 @@ template = "doc.html"
-
-[^1]: More strictly: it proves that he didn't mine it after the other blocks currently on the Bitcoin blockchain. It only gives an ordering within the chain, not a literal timestamp.
-
-[^2]: In addition to planets, there are stars and galaxies. From a PKI perspective they're treated exactly like planets, but on the network they provide infrastructure services like routing.
-
-[^3]: In practice, this will likely be an included service by your sponsoring star. It could be a separate service, but it will be very cheap because it's a commodity and the "cost of production" is very cheap. At any rate, it should never rise above the capital cost of a 400GB hard drive.
-
-[^4]: To extend the metaphor, stars are a neighborhood to live in -- they're easier for other planets to see, so when they want to send you a message they look for your star first. If they don't even know where your star is, they can certainly find your star's galaxy, and that will be enough to locate the star. Of course, if your star is not providing satisfactory service, you can take your planet and move to another star.
+Fancifully, comets are wayward celestial bodies that are great for testing and
+miscellaneous low-value things that won't last for long. Planets are where you
+can build a home and shape it into anything you want it to be. People can always
+find your planet; it's not going anywhere. Moons are useful for special
+purposes, like storage, heavy industry, and anything else you might want to do
+off-planet.[^4]
+
+
+[^0]: Or the equivalent with a hierarchical key structure. In practice, you want
+    to have a master key which signs a junior key for everyday use. You use the
+    master key to rotate the junior key.
+
+[^1]: More strictly: it proves that he didn't mine it after the other blocks
+    currently on the Bitcoin blockchain. It only gives an ordering within the
+    chain, not a literal timestamp.
+
+[^2]: In addition to planets, there are stars and galaxies. From a PKI
+    perspective they're treated exactly like planets, but on the network they
+    provide infrastructure services like routing.
+
+[^3]: In practice, this will likely be an included service by your sponsoring
+    star. It could be a separate service, but it will be very cheap because it's
+    a commodity and the "cost of production" is very cheap. At any rate, it
+    should never rise above the capital cost of a 400GB hard drive.
+
+[^4]: To extend the metaphor, stars are a neighborhood to live in -- they're
+    easier for other planets to see, so when they want to send you a message
+    they look for your star first. If they don't even know where your star is,
+    they can certainly find your star's galaxy, and that will be enough to
+    locate the star. Of course, if your star is not providing satisfactory
+    service, you can take your planet and move to another star.
```

Comment:
```suggestion
<a name="f0" href="#0">[0]</a>: Or the equivalent with a hierarchical key
structure. In practice, you want to have a master key which signs a junior key
for everyday use. You use the master key to rotate the junior key.

<a name="f1" href="#1">[1]</a>: More strictly: it proves that he didn't mine it
after the other blocks currently on the Bitcoin blockchain. It only gives an
ordering within the chain, not a literal timestamp.

<a name="f2" href="#2">[2]</a>: In addition to planets, there are stars and
galaxies. From a PKI perspective they're treated exactly like planets, but on
the network they provide infrastructure services like routing.

<a name="f3" href="#3">[3]</a>: In practice, this will likely be an included
service by your sponsoring star. It could be a separate service, but it will be
very cheap because it's a commodity and the "cost of production" is very cheap.
At any rate, it should never rise above the capital cost of a 400GB hard drive.

<a name="f4" href="#4">[4]</a>: To extend the metaphor, stars are a
neighborhood to live in -- they're easier for other planets to see, so when
they want to send you a message they look for your star first. If they don't
even know where your star is, they can certainly find your star's galaxy, and
that will be enough to locate the star. Of course, if your star is not
providing satisfactory service, you can take your planet and move to another
star.
```